subprojects/packagefiles/sdl3/meson.build

```diff
@@ -0,0 +1,252 @@
+  ['HAVE_FABSF', 'fabsf', '#include<math.h>'],
+  ['HAVE_FLOOR', 'floor', '#include<math.h>'],
+  ['HAVE_FLOORF', 'floorf', '#include<math.h>'],
+  ['HAVE_FMOD', 'fmod', '#include<math.h>'],
+  ['HAVE_FMODF', 'fmodf', '#include<math.h>'],
+  ['HAVE_ISINF', 'isinf', '#include<math.h>'],
+    ['HAVE_ISINFF', 'isinff', '#include<math.h>'],
+# check token ['HAVE_ISINF_FLOAT_MACRO']
+  ['HAVE_ISNAN', 'isnan', '#include<math.h>'],
+  ['HAVE_ISNANF', 'isnanf', '#include<math.h>'],
+# check token ['HAVE_ISNAN_FLOAT_MACRO']
+  ['HAVE_LOG', 'log', '#include<math.h>'],
+  ['HAVE_LOGF', 'logf', '#include<math.h>'],
+  ['HAVE_LOG10', 'log10', '#include<math.h>'],
+  ['HAVE_LOG10F', 'log10f', '#include<math.h>'],
+# check token ['HAVE_LROUND']
+# check token ['HAVE_LROUNDF']
+# check token ['HAVE_MODF']
+# check token ['HAVE_MODFF']
+  ['HAVE_POW', 'pow', '#include<math.h>'],
+# check token ['HAVE_POWF']
+  ['HAVE_ROUND', 'round', '#include<math.h>'],
+# check token ['HAVE_ROUNDF']
+  ['HAVE_SCALBN', 'scalbn', '#include<math.h>'],
+# check token ['HAVE_SCALBNF']
+  ['HAVE_SIN', 'sin', '#include<math.h>'],
+  ['HAVE_SINF', 'sinf', '#include<math.h>'],
+  ['HAVE_SQRT', 'sqrt', '#include<math.h>'],
+  ['HAVE_SQRTF', 'sqrtf', '#include<math.h>'],
+# check token ['HAVE_TAN']
+# check token ['HAVE_TANF']
+# check token ['HAVE_TRUNC']
+# check token ['HAVE_TRUNCF']
+# check token ['HAVE__FSEEKI64']
+# check token ['HAVE_FOPEN64']
+  ['HAVE_FSEEKO', 'fseeko', '#include<stdio.h>'],
+  ['HAVE_FSEEKO64', 'fseeko64', '#include<stdio.h>'],
+# check token ['HAVE_MEMFD_CREATE']
+# check token ['HAVE_POSIX_FALLOCATE']
+  ['HAVE_SIGACTION', 'sigaction', '#include<signal.h>'],
+# check token ['HAVE_SA_SIGACTION']
+# check token ['HAVE_ST_MTIM']
+  ['HAVE_SETJMP', 'setjmp', '#include<setjmp.h>'],
+  ['HAVE_NANOSLEEP', 'nanosleep', '#include<time.h>'],
+  ['HAVE_GMTIME_R', 'gmtime_r', '#include<time.h>'],
+# check token ['HAVE_LOCALTIME_R']
+# check token ['HAVE_NL_LANGINFO']
+  ['HAVE_SYSCONF', 'sysconf', '#include<unistd.h>'],
+  ['HAVE_SYSCTLBYNAME', 'sysctlbyname', '#include<sys/sysctl.h>'],
+  ['HAVE_CLOCK_GETTIME', 'clock_gettime', '#include<time.h>'],
+  ['HAVE_GETPAGESIZE', 'getpagesize', '#include<unistd.h>'],
+# check token ['HAVE_ICONV']
+  ['HAVE_PTHREAD_SETNAME_NP', 'pthread_setname_np', '#include<pthread.h>'],
+  ['HAVE_PTHREAD_SET_NAME_NP', 'pthread_set_name_np', '#include<pthread.h>'],
+  ['HAVE_SEM_TIMEDWAIT', 'sem_timedwait', '#include<semaphore.h>'],
+# check token ['HAVE_GETAUXVAL']
+# check token ['HAVE_ELF_AUX_INFO']
+  ['HAVE_POLL', 'poll', '#include<poll.h>'],
+# check token ['HAVE__EXIT']
+# check token ['HAVE_FCITX']
+  ['HAVE_INOTIFY_INIT1', 'inotify_init1', '#include<sys/inotify.h>'],
+# check token ['HAVE_INOTIFY']
+# check token ['HAVE_LIBUSB']
+# check token ['HAVE_O_CLOEXEC']
+]
+
+foreach f : check_functions
+  if cc.has_function(f.get(1), prefix : f.get(2), dependencies: [sdl_deps])
+    cdata.set(f.get(0), 1)
+  endif
+endforeach
```

Comment:
Early Windows failure since dl can be a Windows emulated one which we happen to provide in the wrapdb. I suppose that isn't a general problem outside of the wrapdb CI... if https://github.com/dlfcn-win32/dlfcn-win32 isn't useful for SDL3 we can just use `allow_fallback: false` for it?